src/test/run-make/issue-26995/Makefile

```diff
@@ -0,0 +1,6 @@
+-include ../tools.mk
+
+all:
+	printf "#[path=\"%s/%s\"] pub mod baz;\n" `pwd` "bar/baz.rs" > foo.rs
```

Comment:
Didn't think of that, but it seems like rustdoc doesn't generate files for empty public modules.

That did make another bug clear though, namely that absolute paths not contained in the root directory are still broken just like before.
(I'm gonna try to put these in a `src/__root__` directory; for example `/dev/null` -> `src/__root__/dev/null.html`)